engine/common/rpc/convert/execution_data.go

```diff
@@ -316,6 +316,39 @@ func messageToTrustedTransaction(
 	return *t, nil
 }
 
+func MessageToRegisterID(m *entities.RegisterID) (*flow.RegisterID, error) {
```

Comment:
the response object is very simple, may as well return by value. This also makes it more consistent with `flow.RegisterIDs`
```suggestion
func MessageToRegisterID(m *entities.RegisterID) (flow.RegisterID, error) {
```